app/models/application_proceeding_type.rb

```diff
@@ -69,12 +71,15 @@ def proceeding_case_p_num
 
   private
 
+  def only_one_lead_proceeding_per_application
+    return if lead_proceeding == false
+
+    apt = legal_aid_application.application_proceeding_types.detect(&:lead_proceeding?)
+    errors.add(:lead_proceeding, :multiple) if apt.present? && apt.id != id
```

Comment:
will this error ever be seen? we dont allow the user to select a lead proceeding so if this error appeared how would they change it?